types/case-sensitive-paths-webpack-plugin/case-sensitive-paths-webpack-plugin-tests.ts

```diff
@@ -0,0 +1,11 @@
+import { Configuration } from 'webpack';
+import * as CaseSensitivePathsWebpackPlugin from 'case-sensitive-paths-webpack-plugin';
```

Comment:
Please use `import x = require('x')` syntax for commonjs exports that use `export = x` syntax.